README.md

```diff
@@ -0,0 +1,89 @@
+# Diplomat
+Diplomat is an experimental Rust tool for generating FFI definitions allowing many other languages to call Rust code. With Diplomat, you can simply define Rust APIs to be exposed over FFI and get high-level C, C++, and JavaScript bindings automatically!
```

Comment:
We should probably mention that this is pluggable and explain the pluggability somewhere